controllers/sriovoperatorconfig_controller.go

```diff
@@ -175,6 +183,12 @@ func (r *SriovOperatorConfigReconciler) syncConfigDaemonSet(dc *sriovnetworkv1.S
 	data.Data["ClusterType"] = utils.ClusterType
 	data.Data["DevMode"] = os.Getenv("DEV_MODE")
 	data.Data["ImagePullSecrets"] = GetImagePullSecrets()
+	if dc.Spec.ConfigurationMode == sriovnetworkv1.SystemdConfigurationMode {
+		data.Data["UsedSystemdMode"] = true
```

Comment:
`data.Data["UsedSystemdMode"] = (dc.Spec.ConfigurationMode == sriovnetworkv1.SystemdConfigurationMode)` ?

Reply:
I will like to leave this one like it's I think is more readable